features/meta.yml

```
@@ -1,6 +1,5 @@
-draft_date: 2024-06-05
 name: <meta>
-description: The `<meta>` element TODO.
+description: The `<meta>` element represents metadata used by the browser or search engines about the page including description, keywords, character sets, etc.
```

Ooh, nice update!

Style nits:

- Keep "about the page" with its referent
- Avoid Latin abbreviations

```suggestion
description: The `<meta>` element represents metadata about the page used by the browser or search engines, including description, keywords, and character sets.
```

Happy to see this merge soon!